src/modules/Dashboard/features/Applet/DashboardAppletSettings/DashboardAppletSettings.types.ts

```diff
@@ -1,6 +1,7 @@
 import { Roles } from 'shared/consts';
 
 export type GetSettings = {
+  appletName?: string;
```

Comment:
Do we need to retain this property? The same is in the BuilderApplestSettings getSettings?